signing/signing.go

```diff
@@ -0,0 +1,50 @@
+package signing
+
+import (
+	"context"
+	"log"
+	"net/http"
+	"path/filepath"
+	"strings"
+
+	"github.com/gorilla/mux"
+	"github.com/ossf/scorecard/v2/cron/data"
+)
+
+type ScorecardOutput struct {
+	JsonOutput string
+}
+
+func GetResults(w http.ResponseWriter, r *http.Request) {
+	// Get params to build GCS filepath.
+	ctx := context.Background()
+	bucketURL := "gs://ossf-scorecard-results"
+	host := mux.Vars(r)["host"]
+	orgName := mux.Vars(r)["orgName"]
+	repoName := mux.Vars(r)["repoName"]
+	resultsFile := filepath.Join(host, orgName, repoName, "results.json")
+
+	// Sanitize input and log query.
+	resultsFile = filepath.Clean(resultsFile)
+	matched, err := filepath.Match("*/*/*/results.json", resultsFile)
+	if err != nil || !matched {
+		http.Error(w, "error verifying filepath format", http.StatusInternalServerError)
+		log.Println("matched filepath?", matched, err)
+		return
+	}
+	resultsFileEscaped := strings.Replace(resultsFile, "\n", "", -1)
+	resultsFileEscaped = strings.Replace(resultsFileEscaped, "\r", "", -1)
+	log.Printf("Querying GCS bucket for: %s", resultsFileEscaped)
+
+	// Query GCS bucket.
+	resultsBytes, err := data.GetBlobContent(ctx, bucketURL, resultsFileEscaped)
+	if err != nil {
+		http.Error(w, "error pulling from GCS bucket", http.StatusInternalServerError)
```

Comment:
Would be nice if you could check that you get the proper error in your unit tests. Maybe you could declare the error as:
```
var errorPullingBucket = errors.New("error pulling from GCS bucket")
...
http.Error(w, errorPullingBucket.Error(), http.StatusInternalServerError)
```

for each of these errors.